manifests/vcenter/dvs_portgroups.pp

```diff
@@ -1,4 +1,18 @@
 # Class that manages DvSwitch Portgroups
+# @summary Manages distributed virtual switch (dvswitch or dvs for short) portgroups.
+#
+# @param dvswitch
+#   Name of the dvswitch we will add portgroups on.
+# @param dvs_pgs
+#   Hash of portgroup and VLAN IDs.
+#
+# @example Basic usage
+#   powercli::vcenter::dvs_portgroups {'my-vcenter.fqdn.tld':
+#     dvswitch => 'my_dvswitch_name_here',
+#     $dvs_pgs => {
+#       'vlan_10_portgroup_name_here' => 10,
+#       'vlan_11_portgroup_name_here' => 11 }
```

Comment:
trailing } should be on its own line with a comm after it.

basically this whole thing should look like:

```
   powercli::vcenter::dvs_portgroups {'my-vcenter.fqdn.tld':
     dvswitch => 'my_dvswitch_name_here',
     dvs_pgs => {
       'vlan_10_portgroup_name_here' => 10,
       'vlan_11_portgroup_name_here' => 11,
      },
   }
```

Reply:
fixed